src/System.Runtime.Analyzers/Core/SpecifyCultureInfo.cs

```diff
@@ -43,6 +43,8 @@ public override void Initialize(AnalysisContext analysisContext)
 
             analysisContext.RegisterCompilationStartAction(csaContext =>
             {
+                var obsoleteAttributeType = WellKnownTypes.ObsoleteAttribute(csaContext.Compilation);
```

Comment:
System.Obsolete might be undefined in compilation. Can you add logic to handle null `obsoleteAttributeType` type? Probably just add a null check at start of WhereMethodDoesNotContainAttribute to return all methods.